RevenueCat/Scripts/PurchasesConfiguration.cs

```diff
@@ -116,6 +118,12 @@ public Builder SetDangerousSettings(DangerousSettings dangerousSettings)
                 return this;
             }
 
+            [Obsolete("RevenueCat currently uses StoreKit 1 for purchases, as its stability in production " +
+                      "scenarios has proven to be more performant than StoreKit 2.\n" +
+                      "We're collecting more data on the best approach, but StoreKit 1 vs StoreKit 2 is \n" +
+                      "an implementation detail that you shouldn't need to care about.\n" +
+                      "We recommend not using this parameter, letting RevenueCat decide for " +
+                      "you which StoreKit implementation to use.", false)]
```

Comment:
And this one turns into this in the code
<img width="1169" alt="Screenshot 2023-03-13 at 3 22 31 PM" src="https://user-images.githubusercontent.com/3922667/224795562-2e831c76-7114-4f30-ae43-131b17196635.png">